Index: fibseq/src/fibseq/fibseq.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package fibseq;\r\nimport java.util.Scanner;\r\n\r\npublic class fibseq {\r\n\tstatic int chosenNbr = 0;\r\n\tstatic int firstNbr = 0;\r\n\tstatic int secondNbr = 1;\r\n\t\r\n    public static void main(String args[]) {\r\n\r\n    \tSystem.out.println(\"How many Fibonacci Numbers?\");\r\n        Scanner scanner = new Scanner(System.in);\r\n        chosenNbr = scanner.nextInt();\r\n    \t\r\n    \t\r\n\t    System.out.print(\"Fib Numbers: 0 1 \");\r\n\t    for(int i = 2; i < chosenNbr; i++){\r\n\t\t\tSystem.out.print(fibCalc(i) +\" \");\r\n    }\r\n\t    scanner.close();\r\n    }\r\n\r\n\r\n\tprivate static int fibCalc(int i) {\r\n\t\t int newNbr = firstNbr + secondNbr;\r\n         firstNbr = secondNbr;\r\n         secondNbr = newNbr;\r\n\t\treturn newNbr;\r\n\t} \r\n}\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fibseq/src/fibseq/fibseq.java b/fibseq/src/fibseq/fibseq.java
--- a/fibseq/src/fibseq/fibseq.java	(revision 105d1249c1a2dffac331fc41eb0187db2fa9dc5b)
+++ b/fibseq/src/fibseq/fibseq.java	(date 1705804186347)
@@ -20,7 +20,14 @@
 	    scanner.close();
     }
 
-
+	/**
+	 * This function calculates numbers in the Fibonacci sequence.
+	 * Outputs next number in the sequence up to the total number chosen by the user.
+	 * @param firstNbr First number for addition to calculate new Fibonnaci Number.
+	 * @param secondNbr Second number for addition to calculate new Fibonnaci Number.
+	 * @param countNbr Counts the number of iterations it's cycling through.
+	 * @param chosenNbr The number provided by the user for desired number of output.
+	 */
 	private static int fibCalc(int i) {
 		 int newNbr = firstNbr + secondNbr;
          firstNbr = secondNbr;
